kaltura/kmc-ng
# Loaders fix

### PR information
**What kind of change does this PR introduce?** (check one with "x")
- [ x] Feature
- [x ] Refactoring (no functional changes, no api changes)

**What is the current behavior?** (You can also link to an open issue here)
Loaders are set as an attribute directive which required all the loader properties to be set as additional attributes on the loaded component definition
No loader for categories and refine

**What is the new behavior?**
Loaders are now components wrapping the loaded content. Properties are now attributes of the loader component only.
Categories and Refine content are now wrapped in loaders

**Does this PR introduce a breaking change?** (check one with "x")
- [x ] No


If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

---

### Other information

**Where should the reviewer start?**
HTML templates for Entries table, Categories and Refine
**How should this be manually tested?**
1. Verify the entries table loader shows correctly
2. Verify entries load errors are displayed correctly with a refresh button (disable network and click refresh)
3. In order to test categories and refine loaders, you must set the loading status to true at all times in the store service.
**Any background context you want to provide?**
